src/main/java/com/google/javascript/clutz/Options.java

```diff
@@ -227,6 +250,18 @@ public CompilerOptions getCompilerOptions() {
         throw new RuntimeException("Error reading goog provides file " + googProvidesFile, e);
       }
     }
+
+    if (knownClassAliasesFile != null) {
+      try {
+        for (String line : Files.readLines(new File(knownClassAliasesFile), UTF_8)) {
+          String[] alias = line.split(",");
```

Comment:
add some basic checking here. If alias.length is != 2, should throw some meaningful error.